### PR TITLE
Add character count command

### PR DESCRIPTION
**Pull request by ubuntudroid**
## Description

Counts the characters of either the clipboard or the passed argument.

## Type of change

- [x] New script command

## Screenshot

CleanShot 2021-06-17 at 16 41 00

## Dependencies / Requirements

Needs python 3.x and the [pasteboard library](https://github.com/tobywf/pasteboard).

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ubuntudroid** [comment]
Here is a gif that better shows the output of the script (the one in the PR description was trimmed to aggressively and I can't change it anymore:

CleanShot 2021-06-17 at 16 53 11

File: commands/developer-utils/count-characters.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+# Dependency: This script requires the python `pasteboard` library: https://github.com/tobywf/pasteboard
+# Install via pip: `pip install pasteboard`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Count characters
```
Comment (unnamedd) [inline commands/developer-utils/count-characters.py:8]:
```suggestion
# @raycast.title Count Characters
```

File: commands/developer-utils/count-characters.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon 🤖
+# @raycast.argument1 { "type": "text", "placeholder": "Text", "optional": true }
+# @raycast.packageName Developer Utils
```
Comment (unnamedd) [inline commands/developer-utils/count-characters.py:14]:
```suggestion
# @raycast.packageName Developer Utilities
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hey @ubuntudroid, thanks for submit your Script Command and welcome aboard. 🎉 

Just two small things to change to keep the conformance with the guideline.

**Review by unnamedd — APPROVED** [review]
Looks good to me! Thanks for the changes @ubuntudroid!

**Review by fe9lix — APPROVED** [review]

**Comment by jaklan** [comment]
@ubuntudroid @unnamedd Dependency on a globally installed pip package is a **very bad** practice. Especially there's absolutely no need to use it, you can just go with plain Bash + Python script with no dependencies, exactly like here:
https://github.com/raycast/script-commands/blob/master/commands/conversions/strikethrough.sh

**Comment by unnamedd** [comment]
@jaklan I didn't say anything about a global pip install, but the point here is the use of `pasteboard` which is clearly not part of the Python framework. Am I wrong?

**Comment by es183923** [comment]
something like the following could replace the script, and it doesn't have any dependencies.
```bash
#!/bin/bash
str=$(pbpaste)
echo ${#str}
```

**Comment by unnamedd** [comment]
> something like the following could replace the script, and it doesn't have any dependencies.
> 
> ```shell
> #!/bin/bash
> str=$(pbpaste)
> echo ${#str}
> ```

For sure, then I’d like to ask one of you to bring this benefit to our community bringing this improvement updating the Script Command! cc @jaklan @es183923

**Comment by es183923** [comment]
Sure!

**Pull request merged**